01-js-syntax/exercise/Syntax,FunctionsAndStatements-Exercise.js

```diff
@@ -0,0 +1,194 @@
+// 01. Fruit
```

Comment:
Great first PR but I would advice you to upload one solution per PR and branch. It will be easier for the reviewers and for you to maintain your solutions. Just pick the ones that you really want to show & discuss.
The idea of this repository is not the use it as a store for everything that we have as a code but to learn from others and evolve as developers.